pkg/testcase/networkconnectivity.go

```diff
@@ -0,0 +1,71 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+// TestInternodeConnectivityMixedOS Deploys services in the cluster and validates communication between linux and windows nodes
+func TestInternodeConnectivityMixedOS(deleteWorkload bool) {
+	_, err := shared.ManageWorkload("apply",
+		"pod_client.yaml","windows_app_deployment.yaml")
+	Expect(err).NotTo(HaveOccurred(), err)
+	
+	assert.ValidatePodIPByLabel([]string{"app=client","app=windows-app"},[]string{"10.42","10.42"})
```

Comment:
What if we're passing in a specific clusterCIDR or serviceCIDR where it's not in the `10.42` range but rather the `10.43` or something? Would this test then fail?

If so, is there a way we could get this value dynamically? Or validate some other part of it?

Reply:
Will check for that option for improvements but atm this is how it can be validated